plugin/clang.vim

```diff
@@ -767,6 +771,9 @@ func! s:ClangCompleteInit(force)
   " add include directories if is enabled and not ow
   if g:clang_include_sysheaders && ! l:is_ow
     let l:incs = s:DiscoverIncludeDirs(g:clang_exec, b:clang_options)
+    if g:clang_has_win
+      let l:incs += s:DiscoverIncludeDirs(g:gcc_exec, b:clang_options)
+    endif
```

Comment:
I'm not sure if `s:DiscoverIncludeDirs` can handle the output of gcc, because I wrote this function only for clang.
Please also paste the output of gcc to the pull request.

Any idea to wrap following code into a new function ?

```viml
    let l:incs = s:DiscoverIncludeDirs(g:clang_exec, b:clang_options)
    if g:clang_has_win
      let l:incs += s:DiscoverIncludeDirs(g:gcc_exec, b:clang_options)
    endif
```